.env.development

```diff
@@ -34,6 +34,6 @@ HOTJAR_VERSION=6
 HOTJAR_DEBUG=''
 ENABLE_SKILLS_QUIZ='true'
 ENABLE_NOTICES=''
-FEATURE_ENABLE_PROGRAMS=''
+FEATURE_ENABLE_PROGRAMS=true
```

Comment:
shouldn't it be a string just like other env variables?

Reply:
Can we also add `FEATURE_PROGRAM_TYPE_FACET` here to make it enabled by default in devstack

Reply:
I have tested it on my system, quotes are not required. and FEATURE_PROGRAM_TYPE_FACET program is already defined in this file.